src/sas_airflow_provider/operators/sas_jobexecution.py

```diff
@@ -71,7 +71,7 @@ def execute(self, context):
         headers = {"Accept": "application/vnd.sas.job.execution.job+json"}
         response = session.post(url, headers=headers, data=payload, verify=False)
 
-        if response.status_code != 200:
+        if(response.status_code < 200 or response.status_code > 300):
```

Comment:
can you change this to >= 300

Reply:
The status code check now will reject 300, as well.